client/ayon_houdini/api/lib.py

```diff
@@ -392,30 +391,29 @@ def parm_values(overrides):
 def reset_framerange(fps=True, frame_range=True):
     """Set frame range and FPS to current folder."""
 
-    project_name = get_current_project_name()
-    folder_path = get_current_folder_path()
-
-    folder_entity = ayon_api.get_folder_by_path(project_name, folder_path)
-    folder_attributes = folder_entity["attrib"]
+    task_entity = get_current_task_entity()
 
     # Set FPS
     if fps:
-        fps = get_folder_fps(folder_entity)
+        fps = get_entity_fps(task_entity)
         print("Setting scene FPS to {}".format(int(fps)))
         set_scene_fps(fps)
 
     if frame_range:
 
         # Set Start and End Frames
-        frame_start = folder_attributes.get("frameStart")
-        frame_end = folder_attributes.get("frameEnd")
+        frame_start = task_entity.get("frameStart")
+        frame_end = task_entity.get("frameEnd")
 
         if frame_start is None or frame_end is None:
-            log.warning("No edit information found for '%s'", folder_path)
+            folder_path = get_current_folder_path()
+            task_name = task_entity["name"]
+            log.warning("No edit information found for '%s' > '%s'",
+                        folder_path, task_name)
             return
 
-        handle_start = folder_attributes.get("handleStart", 0)
-        handle_end = folder_attributes.get("handleEnd", 0)
+        handle_start = task_entity.get("handleStart", 0)
+        handle_end = task_entity.get("handleEnd", 0)
```

Comment:
```suggestion
        attributes = task_entity["attrib"]
        frame_start = attributes.get("frameStart")
        frame_end = attributes.get("frameEnd")

        if frame_start is None or frame_end is None:
            folder_path = get_current_folder_path()
            task_name = task_entity["name"]
            log.warning("No edit information found for '%s' > '%s'",
                        folder_path, task_name)
            return

        handle_start = attributes.get("handleStart", 0)
        handle_end = attributes.get("handleEnd", 0)
```

Reply:
Ah yes, obviously! Thanks. Even, better.... let me kill that folder fallback since it shouldn't be there.